### ISSUE TITLE
Errors when disassembling 

### ISSUE THREAD

**Issue by cubicYYY** [GoSecure/php7-opcache-override#9]
```
Traceback (most recent call last):
  File "./opcache_disassembler.py", line 307, in <module>
    ast = disassembler.create_ast(sys.argv[len(sys.argv) - 1])
  File "./opcache_disassembler.py", line 186, in create_ast
    functions = opcache['script']['function_table']['buckets']
KeyError: 'buckets'
```
command:python3 ./opcache_disassembler.py -c -a64 phpinfo.php.bin  
Tried:changing PHP version(3.4~3.9),run in wsl,but none of them worked.
[phpinfo.php.zip](https://github.com/GoSecure/php7-opcache-override/files/7699003/phpinfo.php.zip)
Here's my file
:)Thanks

**Comment by obilodeau** [comment]
There's probably something that needs to be adapted to support newer PHP version. This was developed for PHP 7.

> Tried:changing PHP version(3.4~3.9),run in wsl,but none of them worked.

You are saying PHP version 3.4 or 3.9 but I assume you are mistaken since PHP 3 was released in 1998.

Unfortunately, we do not plan to maintain our OPCache disassembler. That said, we would accept contributions if you ever manage to understand what is not working properly.

**Comment by andypost** [comment]
Is there plans to allow it parsing PHP 8 files?

**Comment by obilodeau** [comment]
No, this was a proof-of-concept done by a former intern to demonstrate what we thought was problematic behavior with PHP. No work is planned for this project. Contributions would be accepted. Donating the project to another organization or individual would also be possible.
